app/sliders/config_default.yaml

```diff
@@ -0,0 +1,70 @@
+bar:
```

Comment:
I guess when we have specific titles for our charts we might change the ids here to e.G. "bubble_default" and in the config_custom.yaml use a naming like "basisdaten_line_chart" or sth that like. But for now I think that looks pretty neat

Reply:
Yes, definitely! That was the idea :)